fhir-mongoid-models.gemspec

```diff
@@ -3,7 +3,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
 Gem::Specification.new do |spec|
   spec.name = 'fhir-mongoid-models'
-  spec.version = '0.0.1'
+  spec.version = '0.0.2'
   spec.authors = %w[Andrew Bird Ashok Dongare Joseph Kotanchik Rohit Kandimalla Serhii Ilin]
```

Comment:
not related to this but can we make these names single strings. like ['Andrew Bird', 'Ashok Dongare',...]. Currently Andrew Bird is treated as two persons Andrew and Bird. https://rubygems.org/gems/fhir-mongoid-models